lib/Textmaster/Model/Project.php

```diff
@@ -319,6 +320,42 @@ public function addDocuments(array $documents)
     /**
      * {@inheritdoc}
      */
+    public function getPotentialAuthors($status = null)
+    {
+        if (null === $this->getId()) {
+            throw new BadMethodCallException(
+                'The project must be saved before getting authors who can do it.'
+            );
+        }
+
```

Comment:
It's already done here https://github.com/worldia/textmaster-api/blob/master/lib/Textmaster/Api/Project/Author.php#L54.